bin/init-release

```diff
@@ -5,62 +5,56 @@ releaseVersion=$1
 masterTag=$2
 hasTag=$(git tag -l "$masterTag" | wc -l | tr -d '[:space:]')
 
-
-# Helper function to display text in yellow.
-function log () {
-  echo -e "\033[33m"
-  echo "$1"
-  echo -e "\033[0m"
-}
-
+currentDir=$(node -e "console.log(require('path').dirname(require('fs').realpathSync('$BASH_SOURCE')))")
```

Comment:
@marcbachmann could you explain that in detail again? I know the path is a problem, when you use this scripts via a npm installation, but I didn't fully understand why.